packages/frontend/src/components/PieChartConfig/PieChartLayoutConfig.tsx

```diff
@@ -61,10 +59,10 @@ const PieLayoutConfig: React.FC = () => {
     } = useVisualizationContext();
 
     const selectedMetric = useMemo(() => {
-        return [...metrics, ...customMetrics, ...tableCalculations].find((m) =>
```

Comment:
perhaps this should live on the viz provider

Reply:
allMetrics live inside `pieChartConfig` and in an updated config `allMetrics` comes from visualization context